projects/core/modules/notifications/notification-alert/notification-alert-options.ts

```diff
@@ -0,0 +1,22 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiNotification} from '@taiga-ui/core/enums';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface NotificationAlertOptions {
```

Comment:
Please add `hasIcons`, `status` and `hasCloseButton` too

Reply:
Done